packages/ai-jsx/src/lib/openai.tsx

```diff
@@ -567,6 +567,10 @@ export async function* OpenAIChatModel(
     } while (next.value.choices.length == 0);
 
     logger.trace({ deltaMessage: next.value }, 'Got delta message');
+
+    if (next.value.choices[0].finish_reason) {
```

Comment:
I dont quite understand how the loggers are bound to the context.  but does it make more sense for the finish reason to be bound to the conversation, or is the logger bound 1-1 with the thing that `renderToConversation` gets for the output

Reply:
Yup, this will only be set once for each `renderId` that corresponds to a model call. (Similar to the `ai.jsx.prompt`/`ai.jsx.completion` attributes.)

Reply:
Makes sense!

🐑 it